numba/dispatcher.py

```diff
@@ -659,6 +674,11 @@ def __init__(self, py_func, locals={}, targetoptions={},
         functools.update_wrapper(self, py_func)
 
         self.targetoptions = targetoptions
+        if "disable_reflected_list" in py_func.__globals__ and \
+                py_func.__globals__["disable_reflected_list"].__module__ \
```

Comment:
Can this check for the value identity instead of by name?

Reply:
Probably, yes.

Reply:
I don't think this is possible:

```
ipdb> type(py_func.__globals__["disable_reflected_list"].__module__)
<class 'str'>
```